tests/helpers/color.test.ts

```diff
@@ -52,16 +52,18 @@ const testColors: { input: Color; hex: Color; rgba: RGBA; hsla: HSLA }[] = [
     rgba: { a: 1, r: 30, g: 80, b: 16 },
     hsla: { a: 1, h: 107, s: 66.67, l: 18.8 },
   },
+  {
+    input: "rgb(30, 80, 16, 0.5)",
+    hex: "#1E501080",
+    rgba: { a: 0.502, r: 30, g: 80, b: 16 },
+    hsla: { a: 0.502, h: 107, s: 66.67, l: 18.8 },
+  },
 ];
 
 describe("toHex", () => {
   test.each(testColors)("toHex() %s", ({ input, hex }) => {
     expect(toHex(input)).toBe(hex);
   });
-
-  test("toHex with rgba string representation", () => {
```

Comment:
should be tested now that toHex supports "rgba()"

Reply:
added rgba in the test data above :)

Reply:
:blind_emoji: